### ISSUE TITLE
RSS for recently played songs does not show other's people songs

### ISSUE THREAD

**Issue by cquike** [ampache/ampache#3996]
## Description

The recently played songs widget in the home web UI page shows songs played by other users who want to share it. That works fine. However, the RSS feed that you can subscribe from the icon shown in the recently played widget ignores the songs played by other players.

### To reproduce

1. Go to the home web UI for a user1 with the "Show Recently Played" widget enabled in the Home section of the interference preferences.
2. Next to the "recently played" title click on the RSS feed and use a RSS client to subscribe to it.
3. Play a song with user2 that has the option "Share Recently Played information" enabled in the preferences.
4. Check that the song played by user2 shows up in the recently played widget in the web UI.
5. Check that the same song _does not_ show up in the RSS client once the feed is updated.

### Expected behavior
The RSS feed should show the same information as the web widget.

## Environment

* Ampache version: 6.5.0
* Server operating system: Debian 12
* Client operating system: Debian 12

### Client type

* Web

**Comment by lachlan-00** [comment]
The sql gets all users with the preference but doesn't seem to load in the output. so thats possibly requiring the user.

i think it might also be the preference ID.

could you tell me the preferences names in this query?
```
SELECT * from `preference` where `name` like "allow_personal%";
```

**Comment by lachlan-00** [comment]
fixed with 04bfdbcb1

the logic on user > 0 was stopping the has preference from working

**Comment by cquike** [comment]
Just wanted to confirm that indeed that change fixes the issue.
Thanks!
